src/server/scripts/BrokenIsles/zone_mardum.cpp

```diff
@@ -1645,6 +1653,334 @@ class spell_give_me_sight_beyond_sight_periodic : public AuraScript
+    }
+
+    bool OnGossipSelect(Player* player, uint32 menuId, uint32 gossipListId) override
+    {
+        if (menuId == GOSSIP_MENU_USE_SPECTRAL_SIGHT && gossipListId == GOSSIP_OPTION_USE_SPECTRAL_SIGHT)
+        {
+            Talk(SAY_JAYCE_USE_SPECTRAL_SIGHT);
+            player->KilledMonsterCredit(KILLCREDIT_SPEAK_WITH_JAYCE);
+            player->CastSpell(player, SPELL_GIVE_ME_SIGHT_PERIODIC_AURA);
+            player->CastSpell(player, SPELL_GIVE_ME_SIGHT_PERIODIC_DUMMY);
+            CloseGossipMenuFor(player);
+        }
+        return true;
+    }
+};
+
+struct HiddenNoMorePathDelay
+{
+    uint32 PathId;
+    Milliseconds ActionDelay;
+    Milliseconds MoveDelay;
+};
+
+std::unordered_map<std::string_view, HiddenNoMorePathDelay> HiddenNoMorePathMap = {
+    { "CrypticHollowDh01",          { PATH_CRYPTIC_HOLLOW_DEMON_HUNTER_01, 554ms, 2990ms } },
+    { "CrypticHollowDh02",          { PATH_CRYPTIC_HOLLOW_DEMON_HUNTER_02, 270ms, 1690ms } },
+    { "CrypticHollowDh03",          { PATH_CRYPTIC_HOLLOW_DEMON_HUNTER_03, 554ms, 1690ms } },
+    { "CrypticHollowDh04",          { PATH_CRYPTIC_HOLLOW_DEMON_HUNTER_04, 554ms, 2990ms } },
+    { "CrypticHollowDh05",          { PATH_CRYPTIC_HOLLOW_DEMON_HUNTER_05, 402ms, 2990ms } },
+    { "CrypticHollowCoilskar01",    { PATH_CRYPTIC_HOLLOW_COILSKAR_01, 0ms, 1521ms } },
+    { "CrypticHollowCoilskar02",    { PATH_CRYPTIC_HOLLOW_COILSKAR_02, 0ms, 2024ms } },
+    { "CrypticHollowCoilskar03",    { PATH_CRYPTIC_HOLLOW_COILSKAR_03, 0ms, 1521ms } },
+    { "CrypticHollowAshtongue01",   { PATH_CRYPTIC_HOLLOW_ASHTONGUE_01, 0ms, 1521ms } },
+    { "CrypticHollowAshtongue02",   { PATH_CRYPTIC_HOLLOW_ASHTONGUE_02, 0ms, 1521ms } },
+    { "CrypticHollowAshtongue03",   { PATH_CRYPTIC_HOLLOW_ASHTONGUE_03, 0ms, 1521ms } },
+};
+
+constexpr Position JaycePrepareJump = { 1263.948f, 1632.9402f, 102.08563f };
+constexpr Position JayceJumpPosition = { 1245.257f, 1639.049f, 101.223f };
+
+// 96436 - Jayce Darkweaver
+struct npc_jayce_darkweaver_cryptic_hollow_private : public ScriptedAI
+{
+    npc_jayce_darkweaver_cryptic_hollow_private(Creature* creature) : ScriptedAI(creature) { }
+
+    void JustAppeared() override
+    {
+        Player* player = me->GetDemonCreatorPlayer();
+
+        me->RemoveNpcFlag(NPCFlags(UNIT_NPC_FLAG_GOSSIP | UNIT_NPC_FLAG_QUESTGIVER));
+        me->CastSpell(nullptr, SPELL_COSMETIC_EYE_BEAM_01_MALE, false);
+
+        Creature* shivarraObject = GetClosestCreatureWithOptions(player, 25.0f, { .CreatureId = NPC_SHIVARRA_CRYPTIC_HOLLOW, .IgnorePhases = true });
+        if (!shivarraObject)
+            return;
+
+        Creature* shivarraClone = shivarraObject->SummonPersonalClone(shivarraObject->GetPosition(), TEMPSUMMON_MANUAL_DESPAWN, 0s, 0, 0, player);
+        if (!shivarraClone)
+            return;
+
+        ObjectGuid shivarraGuid = shivarraClone->GetGUID();
+
+        _scheduler.Schedule(4s, [this, shivarraGuid](TaskContext task)
```

Comment:
```suggestion
        _scheduler.Schedule(4s, [this, shivarraGuid = shivarraClone->GetGUID()](TaskContext task)
```
this allows us to drop the `ObjectGuid shivarraGuid = shivarraClone->GetGUID();` line